we have to list up all the sub-packages

hiroshi-matsuda-rit opened the pull request:
@izziiyt @kazuma-t 
After some work arounds, I found that package list in setup.py should contain all the sub-packages.

hiroshi-matsuda-rit committed:
we have to list up all the sub-packages

hiroshi-matsuda-rit committed:
debug

izziiyt commented on setup.py:
```diff
@@ -17,7 +17,15 @@ def readme():
       url="https://github.com/WorksApplications/Sudachi",
       license="Apache-2.0",
-      packages=["sudachipy"],
+      packages=[
```
@hiroshi-matsuda-rit Thanks for your adjustments.   
How about using `setuptools.find_namespace_packages` ? In this style, we need to add module name when we add new modules.

hiroshi-matsuda replied:
Sure, it's better. I'll replace it with that soon.

hiroshi-matsuda-rit committed:
refactor with setuptools.find_namespace_packages

hiroshi-matsuda-rit committed:
replace find_namespace_packages to find_packages because pip>=19 is not the majority right now
remove file reference functions causing runtime error while installing from pip
set install_requires as literal list to avoid installing error

hiroshi-matsuda-rit commented:
@izziiyt I applied some fixes to make 'pip install' available.

It looks earlier to use find_namespace_packages because it requires pip>=19.
I replaced it for find_packages.

I removed dynamic file loading functions because it ran while pip installing too and caused 'File Not Found' error.

Please consider to fill the long_description filed with appropriate text.
And we hope your publishing the pip install archives created by the latest setup.py. > @kazuma-t 

Actually, I just published a SudachiPy+SudachiDict package for spaCy Japanese model release from our fork repository for convenience, but I suppose that these archives should be published from owners repository.
https://github.com/megagonlabs/SudachiPy/releases/tag/v0.1.3-dict

hiroshi-matsuda-rit committed:
bump up build version

kazuma-t commented:
@izziiyt After you merge this PR, I will make `.travis.yml` to upload a package to GitHub.
